lib/feedes/document/rss.rb

```diff
@@ -1,16 +1,16 @@
+# frozen_string_literal: true
```

Comment:
RSS1.0 document parser does not implemented yet. 🙇
This module will be implemented by  another pull request.